spec/factories/conditions.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :condition do
+    routing_page { build :page }
+    check_page { nil }
+    goto_page { nil }
+    answer_value { nil }
```

Comment:
I don't know if I should build a more full-featured factory here - the conditions model is relatively simple, so I don't think we need to over-egg the testing in forms-api

Reply:
I think keep it simple like this if you can, we found the factory for form got a bit complicated when we were adding more traits to cover other scenarios